twstock/codes.py

```diff
@@ -24,7 +24,7 @@
 
 def read_csv(path, types):
     global codes, twse, tpex
-    with open(path, newline='') as csvfile:
+    with open(path, newline='', encoding='utf_8_sig') as csvfile:
```

Comment:
In fact, you're right. It seams that the CSV files have no BOM and should not need `utf_8_sig`.